### DISCUSSION TITLE
How about adding a new dbt adapter into dbt-labs?

### DISCUSSION THREAD

**Discussion by hantmac** [dbt-labs/dbt-core#6490] [Community]
Hi dbt-labs, I'm a developer from [databend](https://github.com/datafuselabs/databend) and we have developed an dbt adapter for databend cloud, https://github.com/databendcloud/dbt-databend. And I wrote [this wiki](https://github.com/databendcloud/dbt-databend/wiki/How-to-use-dbt-with-Databend-Cloud) to show how to use dbt with Databend Cloud so the adapter have passed the test example [jaffle_shop](https://github.com/dbt-labs/jaffle_shop) which is a project provided by dbt-lab to demonstrate dbt functionality.

So how about adding this adapter into dbt-labs and we maintain it together? 
Thank you very much!

**Comment by hantmac** [comment]
Add the docs in [this pr](https://github.com/dbt-labs/docs.getdbt.com/pull/2634)

**Comment by dataders** [comment]
Resolved in [this community Slack thread](https://getdbt.slack.com/archives/C030A0UF5LM/p1672551955966329)the answer is that we welcome any and all adapters into the dbt ecosystem, however, there's far too many to feasibly maintain in the dbt-core repo, so now we maintain all seperately, with the exception of postgres.
